Objective-C/CBLQuery.mm

```diff
@@ -275,6 +275,12 @@ - (instancetype) copyWithZone: (NSZone*)zone {
     }
 }
 
+- (NSUInteger) count {
```

Comment:
I think 'count' is ambiguous when it is at the CBLQuery class. Can you rename to columnCount?